content/en/docs/environment variables/_index.md

```diff
@@ -1,16 +1,17 @@
 ---
 title: "Define environment variables"
-linkTitle: "Define environment variables"
+linkTitle: "Environment variables"
 weight: 5
 description: >
     This section describes how to define environment variables.
 aliases:
-- /environment-variables/environment-variables-humanitec
+- /docs/environment-variables/environment-variables-compose/
+- /docs/environment-variables/environment-variables-helm/
 ---
 
 ## Overview
 
-You can pass environment-specific configuration to the container during a deployment. The Score Specification enables a special environment resource type to be used to support such use cases.
+You can pass environment-specific configuration to the container during a deployment. The Score Specification enables a special `environment` resource type to be used to support such use cases.
```

Comment:
Marking `environment` as code to highlight that this is the actual name of the resource type.